### ISSUE TITLE
[CMake] Build type Debug broken

### ISSUE THREAD

**Issue by xmrdsc** [monero-project/monero-gui#2979]
Running with `-DCMAKE_BUILD_TYPE=Debug` will yield the error:

```
make[2]: *** No rule to make target 'monero/external/easylogging++/libeasylogging.a', needed by 'bin/monero-wallet-gui'.  Stop.
```

**Comment by xmrdsc** [comment]
In addition, would be nice if our build bots could build in debug mode when this is fixed.

**Comment by xmrdsc** [comment]
Fixed since #2982
